server/src/testflinger/templates/_agents_table.html

```diff
@@ -49,7 +49,21 @@
             <a href="{{ url_for('testflinger.agent_detail', agent_id=agent.name) }}">{{ agent.name }}</a>
           </td>
           <td data-heading="Location">{{ agent.location or "unknown" }}</td>
-          <td data-heading="State">{{ agent.state }}</td>
+          <td class="has-overflow" data-heading="State">
+            {% if agent.get('comments') %}
+              <span>{{ agent.state }}</span>
+              <span class="p-tooltip--top-left" aria-describedby="tooltip">
+                <i class="p-icon--information"></i>
+                <span class="p-tooltip__message"
+                      role="tooltip"
+                      id="tooltip"
+                      style="white-space: nowrap;
+                             text-align: left">{{ agent.comments }}</span>
+              </span>
```

Comment:
I think we have other tooltips (or may want to add more), could you test if we can use an `id` that is a little more unique?

```suggestion
              <span class="p-tooltip--top-left" aria-describedby="state-tooltip">
                <i class="p-icon--information"></i>
                <span class="p-tooltip__message"
                      role="tooltip"
                      id="state-tooltip"
                      style="white-space: nowrap;
                             text-align: left">{{ agent.comments }}</span>
              </span>
```